cocotb/regression.py

```diff
@@ -320,6 +326,12 @@ def _result_was():
 
         else:
             self.log.error("Test Failed: " + _result_was(), exc_info=exc_info)
+            if _pdb_on_exception:
+                if sys.version_info >= (3, 5):
```

Comment:
How that we dropped Python 2 support this condition can be removed (or you leave it in for a follow-up cleanup, as you like)

Reply:
Let's do this in a follow-up, it will be easier to review all at once I think